.goreleaser.yml

```diff
@@ -10,19 +12,20 @@ builds:
   - binary: listmonk
     main: ./cmd
     goos:
-      - windows
-      - darwin
+      # - windows
+      # - darwin
       - linux
-      - freebsd
-      - openbsd
-      - netbsd
+      # - freebsd
+      # - openbsd
+      # - netbsd
```

Comment:
Just seeing that now that it all runs fine as you have indeed removed machine info, that's good. But for the sake of clarity the reason I commented it: There's no benefit at all in my opinion wasting resources on anything but linux builds because as far as I've noticed and seen in this project no one would benefit of that.

The reason we needed multi arch for listmonk was CPU architecture.

Since you removed the machine info now it builds on all those platforms, which is nice, but there's no benefit in building those.

In fact: If, in the future, you'd need again something more nix-Specific you'll have to maintain the problems of having e.g. a IMO usless windows build which also unnecessarily increases build time.

Any reason or users that actually benefit of building those? AFAIK no one was using that on other platforms and also no one was asking for it hence adding potentially enlarged maintenance and increased build time isn't a benefit.

Agree?

Reply:
> Since you removed the machine info now it builds on all those platforms, which is nice, but there's no benefit in building those.

Hm, why do you think so? It's just one string that was displayed through the API. Before the multi-arch build, we did not have "packaged" binaries for specific architectures other than x86. Surely there are benefits for building multi-arch binaries for people that need it.

> Since you removed the machine info now it builds on all those platforms, which is nice, but there's no benefit in building those.

Yeah, this has been discussed and we'll be removing a few of the unnecessary OS builds from future releases.

> AFAIK no one was using that on other platforms and also no one was asking for it

There have been people asking for Linux builds, and in particular, for ARM. And so we added a few other architectures/OSes too alongside that, but as I said in the statement before, quite a few of those will be removed from future releases since the current releases page induces a lot of clutter, and those can either be built manually by users or added in later releases by us as the demand increases.

Reply:
I think you misunderstand me. I was talking about platforms not arch. I was one of the ones asking for arch and working on it. CPU Architecture, that's what ARM is. And that's important. I was only talking about OS when I meant platform. Sorry for the confusion. So I was only pointing to remove OSes.

Reply:
Correct, I also got confused for a minute there, sorry. We'll be removing some of the OSes/Arch from the goreleaser matrix to bring down the build time in the next release. Certainly, Windows and a few of the *BSD ones aren't needed for now.

Reply:
There was only `linux` before so I'd stick with that for now.